lisa/sut_orchestrator/azure/common.py

```diff
@@ -368,3 +383,70 @@ def wait_copy_blob(
         raise LisaException(f"wait copying VHD timeout: {vhd_path}")
 
     log.debug("vhd copied")
+
+
+def get_data_disk_size(
+    node_features: search_space.SetSpace[str], data_disk_iops: int
+) -> int:
+    if DiskType.DISK_EPHEMERAL in node_features:
+        raise LisaException(f"{DiskType.DISK_EPHEMERAL} only can be OS disk type.")
+    elif DiskType.DISK_PREMIUM in node_features:
+        return DataDisk.get_size(DiskType.DISK_PREMIUM, data_disk_iops)
+    elif DiskType.DISK_STANDARD_HDD in node_features:
+        return DataDisk.get_size(DiskType.DISK_STANDARD_HDD, data_disk_iops)
+    elif DiskType.DISK_STANDARD_SSD in node_features:
+        return DataDisk.get_size(DiskType.DISK_STANDARD_SSD, data_disk_iops)
+    else:
+        raise LisaException("No data disk feature present.")
+
+
+class DataDisk:
+    # refer https://docs.microsoft.com/en-us/azure/virtual-machines/disks-types
+    IOPS_SIZE_PREMIUM: Dict[int, int] = {
```

Comment:
If it's two level dict, the code will be simpler.

iops_mapping:Dict[DiskType, Dict[int, int] = {
    IOPS_SIZE_PREMIUM: {
       120: 4,
       240:64,
       ...
   }
}